src/sidebar/panel.html

```diff
@@ -42,13 +45,24 @@
 
     <div id="footer-buttons">
       <button id="enable-sync"></button>
-      <a target="_blank" id="give-feedback"></a>
+			<div class="wrapper">
+				<button id="demo-menu-top-right"
+        class="mdl-js-button">
+					<i class="material-icons">more_vert</i>
```

Comment:
Sorry, I'm not super familiar w/ Material, but what is "more_vert" here? I imagine it'd possibly cause some issues for screen readers, unless the CSS is tweaking a bunch of ARIA settings or something.